README.md

```diff
@@ -162,37 +159,11 @@ $ hmpb interpret -e election.json -f \
         ]
       },
       "marks": [
-        {
-          "type": "candidate",
-          "contest": "us-senate",
-          "option": "john-cornyn",
-          "score": 0.005037783375314861,
```

Comment:
This very low score is now zero, which is why it's completely removed from the output.